loading_bay_detector.py

```diff
@@ -22,19 +16,13 @@ def run_detector(self):
 
         img = cv2.GaussianBlur(img, (3, 3), cv2.BORDER_DEFAULT)
         hsv = cv2.cvtColor(img, cv2.COLOR_BGR2HSV)
+
         mask = cv2.inRange(hsv, self.calibrator.low_green, self.calibrator.high_green)
         contours_return = cv2.findContours(mask, cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE)
-        contours = None
-        if len(contours_return) == 3:
-            contours = contours_return[1]
-        else:
-            contours = contours_return[0]
+        contours = contours_return[len(contours_return) == 3]
```

Comment:
What is this doing?

Reply:
We wanted to make the if statement more concise. Should we add a comment to describe it or revert it back to the full form?

Reply:
Let's use the full form. The boolean to int conversion is very non-obvious.